### DISCUSSION TITLE
Real IP listeners not showing

### DISCUSSION THREAD

**Discussion by DJMic** [AzuraCast/AzuraCast#7533] [Q&A] [answered]
Hi,
Since several weeks, impossible to see the ip address of listeners. Any idea to resolve this ?
Could it be possible that this would be due to the use of Nginx Proxy Manager ?
I use stable version 0.20.1

![image](https://github.com/user-attachments/assets/22773d25-c82e-4a69-9a11-145afed1bc96)

**Comment by gAlleb** [comment]
<img width="650" alt="Снимок экрана 2024-11-01 в 14 12 23" src="https://github.com/user-attachments/assets/7ec3d1de-b78c-43f4-aa68-4a498b387c83">

**Reply by DJMic** [reply]
I have already try this but it doesn't resolve.

**Reply by gAlleb** [reply]
And even after restarting Azuracast Nginx? 
Hm..

**Reply by DJMic** [reply]
It doesn't resolve. But I think I have a problem with my station.

**Comment by DJMic** [comment]
After various tests, I think I have found why I cannot display the listeners' IP address. When I customize the mount point URL, the listeners IP is not displayed. If I leave the mount point url set to default by azuracast, the IP address displays correctly. How can I resolve this ?

**Comment by DJMic** [comment]
After various tests, I think I have found why I cannot display the listeners' IP address. When I customize the mount point URL, the listeners IP is not displayed. If I leave the mount point url set to default by azuracast, the IP address displays correctly. How can I resolve this ?

**Reply by BusterNeece** [reply]
Huh, if you can reliably reproduce that this is causing the error, then I'd say absolutely create a GitHub Issue about it so we can track it down and figure it out.

**Reply by DJMic** [reply, marked as answer]
I was able to find the source of my problem. I use Nginx Proxy Manager to redirect my stream URL and the listeners connect to this URL. I will instead use the URL generated by Azuracast (https://stream.example.com/listen/mystation/radio.mp3). With this one, no worries to get the IP from the listeners.

**Reply by Jorgfromgermany** [reply]
Hi Folks! I have exactly the same problem. I'm not using the proxy, but I only get "Internal/Reserved IP". I remember that it worked well after installing Azuracast on the Linode. I'm running it using a subdomain, and neither the public page nor the embedded scripts on the homepage deliver the real IPs for the MaxMind GeoLite2.
Do you have any suggestions? Thank you very much for your patience!
